tonic-build/src/prost.rs

```diff
@@ -27,6 +27,7 @@ pub fn configure() -> Builder {
         emit_package: true,
         protoc_args: Vec::new(),
         include_file: None,
+        emit_rerun_if_changed: std::env::var_os("CARGO").is_some(),
```

Comment:
Not sure if run-time detection of whether the code is run as a build script is
1. right, and
2. necessary

We could also just set it to `true` by default.

Reply:
I skimmed through the [cargo reference](https://doc.rust-lang.org/cargo/reference/build-scripts.html) but did not find anything else. I also failed to find results for how to detect Rust code is being run from a build script through Google. I hope this does a good enough job.